starter-code/css/main.css

```diff
@@ -6,3 +6,150 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+    font-family: 'Montserrat', 'sans-serif';
+}
+
+nav {
+    position: fixed;
+    border-color: white;
+    overflow: visible; 
+}
+
+header .spotify-green-logo {
+    float: left;
+    height: 15%;
+    width: 15%;
+    margin-right: 65em;
+}
+
+header ul li {
+    display: inline-block;
+    list-style-type: none;
+    margin: 20px -100px 20px -180px;
+    padding-right: 2em;
+    font-size: 20px;
+    text-align: center;
+}
+
+.banner {
+    text-align: center;
+    color: white;
+    position: relative;
+    background-image: url(../images/landing.jpg);
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    height: 500px;
+    padding-top: 250px;
+}
+
+.banner>h1{
+    font-size: 80px;
+}
+
+.banner>p{
+    font-size: 25px;
+}
+
+.whatson>h2 {
+    border-bottom: 4px solid #1db954;
+    width: fit-content;
+    text-align: center;
+    margin: 40px auto 90px auto;
+    font-size: 35px;
+    font-weight: 1200;
+}
+
+.music-icon {
+    float: left;
+    width: 33.33%;
+    text-align: center;
+}
+
+.image-icon {
+    width: 120px;
+    height: 120px;
+}
+
+.hd-music {
+    float: left;
+    width: 33.33%;
+    text-align: center;
+}
+
+.high-quality-icon {
+    width: 120px;
+    height: 120px;
+}
+
+.stream {
+    float: left;
+    width: 33.33%;
+    text-align: center;
+}
+
+.devices-icon {
+    width: 120px;
+    height: 120px;
+}
+
+.column {
+    margin-bottom: 120px;
+    font-size: 22px;
+}
+
+h3 {
+    color: #1db954;
+    font-weight: bolder;
+}
+
+.column:after {
+    content: "";
+    display: table;
+    clear: both;
+}
+
+#green-section {
+    color: white;
+    background-color: #1db954;
+    margin-right: 5em ;
+    margin-left: 5em ;
+    height: 700px;
+}
+
+#green-section>.text-kanye {
+    position: relative;
+    display: inline-block;
+    width: 40%;
+    text-align: left;
+    font-size: 20px;
+    margin-left: 65px;
+}
+
+#yeezy {
+    border-bottom: 4px solid white;
+    width: fit-content;
+    margin-bottom: 50px;
+}
+
+.text-kanye.h2 {
+    font-weight: bolder;
+}
+
+#green-section>.spotify-app {
+    position: relative;
+    float: right;
+    margin-top: 100px;
+    margin-right: 65px;
+    width: 27%;
+    height: 80%;
+}
+
+#green-section>.spotify-white-logo {
+    position: absolute;
+    margin-top: 20%;
+    height: 20%;
+    width:12%;
```

Comment:
I'm seeing some issues with aspect ratio for your images.  If you remove the `width` properties here and for the `spotify-app` photo, the the size will be adjusted keeping the aspect ratio intact.